src/_pytest/logging.py

```diff
@@ -183,6 +221,12 @@ def add_option_ini(option, dest, default=None, type=None, **kwargs):
         default=DEFAULT_LOG_DATE_FORMAT,
         help="log date format as used by the logging module.",
     )
+    add_option_ini(
+        "--log-auto-indent",
+        dest="log_auto_indent",
+        default=None,
+        help="auto indent multiline messages passed to the logging module. accepts true|on, false|off or an integer",
```

Comment:
I'm not sure we need to support `on/off`... any reason for it?

Reply:
To confirm, you mean only support true/false and remove support for on/off?

Reply:
No specific need to support on/off. If you'd like me to remove it, let me know.

Reply:
> To confirm, you mean only support true/false and remove support for on/off?

Exactly, sorry I wasn't clear. Let's remove it, no need to have support for two set of options. 👍 

Thanks!

Reply:
I ended up taking a different route with respect to this discussion. I discovered that there is a ``strtobool`` method in pytest's config module, which I simply imported and reused for this purpose, which means I don't need to hard-code the accepted values for the strings, and it will simply accept the same strings as all the config options. 

It also happens to be that the standard for all pytest config options is to accept y/n, yes/no, t/f, true/false, on/off and 1/0, so now this option will follow that standard as well.

Reply:
Great, good call